### DISCUSSION TITLE
macOS Ventura 13.0.1 broken dependencies on libsoup3.0

### DISCUSSION THREAD

**Discussion by Mattoje** [Homebrew/discussions#4031] [Everyday usage] [answered]
I'm trying to compile https://github.com/transmission-remote-gtk/transmission-remote-gtk and it fails with this error:

pkg-config --cflags libsoup-3.0
Package icu-uc was not found in the pkg-config search path.
Perhaps you should add the directory containing `icu-uc.pc'
to the PKG_CONFIG_PATH environment variable
Package 'icu-uc', required by 'libpsl', not found

ofc it i add PKG_CONFIG_PATH="/usr/local/opt/icu4c/lib/pkgconfig/" it works but i think this is cleary a misconfiguration or some configuration file missing

**Comment by carlocab** [comment, marked as answer]
This is expected behaviour, because `icu4c` is keg-only. From `brew info icu4c`:
```
icu4c is keg-only, which means it was not symlinked into /usr/local,
because macOS provides libicucore.dylib (but nothing else).

If you need to have icu4c first in your PATH, run:
  echo 'export PATH="/usr/local/opt/icu4c/bin:$PATH"' >> ~/.zshrc
  echo 'export PATH="/usr/local/opt/icu4c/sbin:$PATH"' >> ~/.zshrc

For compilers to find icu4c you may need to set:
  export LDFLAGS="-L/usr/local/opt/icu4c/lib"
  export CPPFLAGS="-I/usr/local/opt/icu4c/include"

For pkg-config to find icu4c you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/icu4c/lib/pkgconfig"
```
